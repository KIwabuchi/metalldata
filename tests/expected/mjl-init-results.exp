{"_state":{"metall_location":"/PATH/TO/DATASTORE/m_results"}}
