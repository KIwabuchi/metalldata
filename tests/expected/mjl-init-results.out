{"_state":{"metall_location":"/tmp/m_results"}}
